docs/xcodecachetotal.md

```diff
@@ -46,6 +46,7 @@ The maximum size limits, for both the JIT code and data caches, that are in use
 
 ## See also
 
+- [`-XX:codecachetotalMaxRAMPercentage`](xxcodecachetotalmaxrampercentage.md)
```

Comment:
Above we have to mention about the percentage option as well:

The default size is 256 MB for a 64-bit VM and 64 MB for a 31/32-bit VM, but no more than 25% of the physical memory available to the VM process,